Makefile

```
@@ -0,0 +1,5 @@
+ROOT:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+IMAGE_NAME:=jenkins/jnlp-slave
```

I am afraid it may impact the local Docker cache if the image is used to start agents

```suggestion
IMAGE_NAME:=jenkins4eval/jnlp-slave:test
```

---

Maybe we even want to include BRANCH_NAME or whatever to avoid collisions between parallel builds